api/src/deployment.rs

```diff
@@ -8,14 +9,16 @@ use rocket::tokio;
 use rocket::Data;
 use shuttle_common::{
     project::ProjectName, DeploymentApiError, DeploymentId, DeploymentMeta, DeploymentStateMeta,
-    Host, Port,
+    Host, LogItem, Port,
 };
 use shuttle_service::loader::{Loader, ServeHandle};
+use shuttle_service::logger::Log;
 use std::collections::HashMap;
 use std::fs::DirEntry;
 use std::io::Write;
 use std::net::{Ipv4Addr, SocketAddr, SocketAddrV4, TcpListener};
 use std::path::{Path, PathBuf};
+use std::sync::mpsc::SyncSender;
```

Comment:
No, it is actually a disadvantage since we can no longer `await`. But I am unable to `await` when sending items into the channel [here](https://github.com/shuttle-hq/shuttle/pull/158/files#diff-356b91dce2079754cf9bec736c38f004010220073ce7d21b045eb5cbd6034303R38-R44) because the trait method is not `async`